spec/09-lexical-structure.md

```diff
@@ -440,9 +440,12 @@ decimal-literal::
   decimal-literal digit
 
 octal-literal::
-  '0'
+  octal-prefix octal-digit
```

Comment:
It seems like a plain `0` now doesn't match any of the rules anymore?

Reply:
Plain `0` is included in `octal-prefix`.

Reply:
It is, but it now has to be followed by octal-digit, so only 00 would be valid, not plain 0.

Reply:
Oh, I see what you’re saying. Under what circumstances should a single digit `0` be interpreted as octal and not decimal?

Reply:
(Regardless of my question, I think removing the plain `0` by itself was an oversight on my part and not intentional.)

Reply:
For 0 the decimal and octal interpretation is the same, so it doesn't really matter which it is, as long as it's one of them. In the grammar it's probably easier to express as octal.